parser.js

```diff
@@ -20,29 +20,32 @@
 
 var farmhash = require('farmhash');
 
-var types = {
-	req_complete_message: 0x01,
-	req_message_fragment: 0x02,
-	req_last_fragment: 0x03,
-	res_complete_message: 0x80,
-	res_message_fragment: 0x81,
-	res_last_fragment: 0x82,
-	res_error: 0xC0
-};
-
-var states = {
-	read_type: 1,
-	read_id: 2,
-	read_seq: 3,
-	read_arg1len: 4,
-	read_arg2len: 5,
-	read_arg3len: 6,
-	read_csum: 7,
-	read_arg1: 8,
-	read_arg2: 9,
-	read_arg3: 10,
-	error: 255
-};
+/* jshint camelcase:false */
+
+var types = {};
+types.reqCompleteMessage = types.req_complete_message = 0x01;
```

Comment:
Let's remove the snake case in a seperate PR as its a breaking change.

It seems unfitting to make a breaking change in a cleanup/jshint PR.